core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3120,6 +3120,19 @@ public String toString() {
               + "`alluxio.client.file.cache.LRUCacheEvictor`.")
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_CLIENT_CACHE_DIR =
+      new Builder(Name.USER_CLIENT_CACHE_DIR)
+          .setDefaultValue("/tmp")
```

Comment:
maybe something like `/tmp/alluxio_cache` to avoid creating too many dirs under `/tmp` directly?

Reply:
Done.